package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "serialize-javascript",
-  "version": "1.4.0",
+  "version": "1.5.0",
```

Comment:
you can remove this change, i can do it when its published

Reply:
Better to do it before it's published :) I have reverted this back now